docs/Developer-Guide.md

```diff
@@ -587,10 +587,15 @@ $ sudo kata-monitor
 
 #### Connect to debug console
 
-Command `kata-runtime exec` is used to connect to the debug console.
+You need to start a container for example:
+```bash
+$ sudo ctr run  --runtime io.containerd.kata.v2 -d docker.io/library/ubuntu:latest testdebug
+```
+
+Then, you can use the command `kata-runtime exec` to connect to the debug console.
```

Comment:
I wonder if adding that the sandbox id is the parameter needed would help with clarity here?
```suggestion
Then, you can use the command `kata-runtime exec <sandbox id>` to connect to the debug console.
```

Reply:
@stevenhorsman thanks for the feedback, changes applied